.github/workflows/rust.yml

```diff
@@ -98,5 +99,14 @@ jobs:
       # it checks for things like memories, exports, imports, available capabilities, and non-determinism
       - name: Check cosmwasm file
         run: cosmwasm-check artifacts/cw_ics08_wasm_eth.wasm
+      - name: Check binary has not changed
+        run: |
+          if ! git diff --quiet HEAD; then
+            echo "Git diff found in the selected directory. Failing the job."
```

Comment:
This checks the whole repo. Might make sense to target it to the wasm file

Reply:
I was considering that, but it there shouldn't be any git diffs in any case at this point. This way we don't have to maintain that path in case we ever move it.

Reply:
Had to remove it. Appears to be non-deterministic across OS'es.
Would have to do the gzip inside a docker container or something. Not going to do that now.

Reply:
Really want to have this, so figure out the solution: check the actual .wasm file instead, which is built deterministically.